src/main/scala/com/gu/liveappversions/Config.scala

```diff
@@ -0,0 +1,39 @@
+package com.gu.liveappversions
+
+import java.io.ByteArrayInputStream
+import java.nio.charset.StandardCharsets
+
+import com.turo.pushy.apns.auth.ApnsSigningKey
+
+object Config {
+
+  case class Env(app: String, stack: String, stage: String) {
+    override def toString: String = s"App: $app, Stack: $stack, Stage: $stage\n"
+  }
+
+  object Env {
+    def apply(): Env = Env(
+      Option(System.getenv("App")).getOrElse("DEV"),
+      Option(System.getenv("Stack")).getOrElse("DEV"),
+      Option(System.getenv("Stage")).getOrElse("DEV"))
+  }
+
+  case class AppStoreConnectConfig(teamId: String, privateKeyId: String, issuerId: String, privateKey: ApnsSigningKey, appleAppId: String)
+
+  object AppStoreConnectConfig {
+    def apply(): AppStoreConnectConfig = {
+      val teamId = System.getenv("APPSTORE_TEAM_ID")
```

Comment:
Right now everything comes from an environment variable - I'll probably migrate this to SSM later.